TrilinosCouplings: Build error (missing header file)

## Bug Report
@trilinos/trilinoscouplings 

### Description
```
.../Trilinos/packages/trilinoscouplings/examples/fenl/fenl_ensemble.hpp:156:18: error: ‘PseudoBlockCGSolMgr’ in namespace ‘Belos’ does not name a template type
     const Belos::PseudoBlockCGSolMgr<Sc, V, O>* cg_solver =
                  ^
.../Trilinos/packages/trilinoscouplings/examples/fenl/fenl_ensemble.hpp:158:9: error: ‘cg_solver’ was not declared in this scope
     if (cg_solver != 0)
         ^
```

--- a/packages/trilinoscouplings/examples/fenl/fenl_ensemble.hpp
+++ b/packages/trilinoscouplings/examples/fenl/fenl_ensemble.hpp
@@ -52,6 +52,7 @@
 
 #if defined( HAVE_STOKHOS_BELOS )
 #include "Belos_TpetraAdapter_MP_Vector.hpp"
+#include "BelosPseudoBlockCGSolMgr.hpp"
 #endif
 
 #if defined( HAVE_STOKHOS_MUELU )

